sdks/python/apache_beam/yaml/yaml_join.py

```diff
@@ -178,26 +178,51 @@ def _SqlJoinTransform(
     fields: Optional[Dict[str, Any]] = None):
   """Joins two or more inputs using a specified condition.
 
+  For example::
```

Comment:
remove extra colon

Reply:
The extra colon is to signify the beginning of a code block.